frontend/tests/e2e/login/login_with_login_gov.spec.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-console */
+import { test, expect } from '@playwright/test';
+import { authenticator } from 'otplib';
+import * as dotenv from 'dotenv';
+import * as path from 'path';
+
+// Load environment variables from frontend/.env.local
+dotenv.config({ path: path.resolve(__dirname, '../../../.env.local') });
+
+// Validate required env variables
+const email = process.env.LOGIN_EMAIL;
```

Comment:
can we make this env var names more specific? ex:
* PLAYWRIGHT_TEST_USER_EMAIL
* PLAYWRIGHT_TEST_USER_PASSWORD
* PLAYWRIGHT_TEST_USER_MFA_KEY